Formula/rio.rb

```diff
@@ -0,0 +1,27 @@
+class Rio < Formula
+  desc "Hardware-accelerated GPU terminal emulator powered by WebGPU"
+  homepage "https://raphamorim.io/rio/"
+  url "https://github.com/raphamorim/rio/archive/refs/tags/v0.0.7.tar.gz"
+  sha256 "5a3fcdc6bbad3274286bc41c5225c9ae576ffcd13b80649247656c0f41422de9"
+  license "MIT"
+  head "https://github.com/raphamorim/rio.git", branch: "main"
+
+  depends_on "pkg-config" => :build
+  depends_on "rust" => :build
+  depends_on "fontconfig"
+  depends_on "freetype"
+  depends_on :macos
+
+  def install
+    system "cargo", "install", *std_cargo_args(path: "rio")
+  end
+
+  test do
+    assert_match version.to_s, shell_output("#{bin}/rio --version")
+    # This test does pass locally for x86 but it fails for containers (Github CI)
+    # which is the case of x86 in the CI
+    if Hardware::CPU.arm?
+      system bin/"rio", "-e", "echo 1; exit"
```

Comment:
this test runs out fine in my local.

Reply:
yea for my both macos (x86 and arm) also does work but for x86 it will always fail in the GH runner due to the container strategy.

Reply:
I see, this failed for x86 container. :(